src/test/java/org/opentripplanner/middleware/OtpMiddlewareTest.java

```diff
@@ -29,7 +32,19 @@ public static void setUp() throws RuntimeException, IOException {
 
         LOG.info("Starting server");
         OtpMiddlewareMain.inTestEnvironment = true;
-        OtpMiddlewareMain.main(new String[]{"configurations/test/env.yml"});
+        // If in the e2e environment, use the secret env.yml file to start the server.
+        // TODO: When ran on Travis CI, this file will automatically be setup.
+        String[] args = getBooleanEnvVar("RUN_E2E")
+            ? new String[] { "configurations/default/env.yml" }
+            : new String[] { "configurations/test/env.yml"};
```

Comment:
The unit tests fail out the box for me. Are there other parameters needed in `test/env.yml` besides `DEFAULT_USAGE_PLAN_ID`?

Reply:
As far as I'm aware, no more than what is (now) defined in the PR description.

Reply:
@binh-dam-ibigroup, you're probably missing `DISABLE_AUTH`.